News_App/src/app/authentication/authentication.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="auth auth__wrapper">
+  <div class="auth" *ngIf="auth.user$ | async as user">
+    <span class="auth__status"> Вы авторизованы как {{ profile.name }}</span>
+    <div>
+      <img
+        class="auth__img"
+        [src]="profile.picture"
+        [alt]="valueAttributeImgAlt"
+      />
+    </div>
+  </div>
+  <div *ngIf="auth.isAuthenticated$ | async">
+    <button class="auth-btn" (click)="logout()">Выйти</button>
+  </div>
+  <div *ngIf="(auth.isAuthenticated$ | async) === false">
```

Comment:
The same like above comment. And any logic should be in your compoment. You can do this compare in component

Reply:
corrected